client/src/components/ClassCard/ClassCard.js

```diff
@@ -103,7 +103,7 @@ export default function SimpleCard(props) {
             Class Code: {Class.code}
           </Typography>
         </div>
-        <DeleteIcon color='error' className={classes.delete} onClick={() => deleteClass(Class.code, Class.instructor?.email, setClasses)}/>
+        <DeleteIcon color='error' className={classes.delete} onClick={() => deleteClass(Class.code, Class.instructor?.email, setClasses, admin)}/>
```

Comment:
In this design delete icon can also work as leave option for students. Please add this feature.

Reply:
@aavishkarmishra : Yes adding as part of #28

Reply:
@aavishkarmishra : Done, Please check new PR